README.md

```diff
@@ -109,24 +109,49 @@ To show some simple examples (from
 [cypress/integration/query.spec.js](cypress/integration/query.spec.js) or [cypress/integration/find.spec.js](cypress/integration/find.spec.js)):
 
 ```javascript
-cy.queryByText('Button Text').should('exist')
-cy.queryByText('Non-existing Button Text').should('not.exist')
-cy.queryByLabelText('Label text', {timeout: 7000}).should('exist')
+cy.queryAllByText('Button Text').should('exist')
```

Comment:
I'm not sure I have a good reason for changing these. It is part of the questions of the pull request. I think `*By` and `*AllBy` make sense for `@test-library/dom`, `@testing-library/react`, and company, but I'm not sure it does for Cypress.

Personally, I recommend always using the `*ByAll` since that's what native Cypress commands do now, but I'm welcoming other opinions